packages/store/src/patterns/patternMatchers.js

```diff
@@ -1547,7 +1547,7 @@ const makePatternKit = () => {
       },
       returns: (returnGuard = UndefinedShape) =>
         harden({
-          klass: 'methodGuard',
+          klass: /** @type {const} */ ('methodGuard'),
```

Comment:
What does `@type {const}` do?

Reply:
Without it, the object is typed `klass: string`. This was it's literally `klass 'methodGuard'` which is tested for in other places statically.